pkg/kubeinteraction/secrets.go

```diff
@@ -97,7 +97,10 @@ func (k Interaction) GetSecret(ctx context.Context, secretopt GetSecretOpt) (str
 	if err != nil {
 		return "", err
 	}
-	return string(secret.Data[secretopt.Key]), nil
+	// a lot of people have problem with this secret, when encoding it to base64 which add a \n when we do :
+	// echo secret|base64 -w0
+	// so cleanup, if someone wants to have a \n or a space in the secret, well then they can't :p
+	return strings.TrimSpace(string(secret.Data[secretopt.Key])), nil
```

Comment:
I don't think we really should modify secret, just for the sake if people wanted to have space or /n into it, we can maybe detect it and output an error in the logs if there is a \n ?

Reply:
okay. 
i faced issue with this with webhook client token
which we use for initialize client
wasn't sure how to remove the \n from secret

Reply:
I did some more research and i think the "internet" concurs it's a bad idea, i created #671  to address a warning which should give a hint to the user..

Reply:
i have removed it for now
pr is good for final review :)